app/controllers/organisation_member/requests_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class OrganisationMember::RequestsController < OrganisationMember::BaseController
+  before_action :set_request, only: :show
+
+  # GET /requests
+  def index
+    @requests = @organisation.requests
+  end
+
+  def new
+    @request = Request.new(requester: @organisation)
+  end
+
+  def create
+    @request = Request.new(document_type: request_params[:document_type],
+                           note: request_params[:note],
+                           requester: @organisation,
+                           requestee: User.find_or_invite(request_params[:requestee_email])
```

Comment:
There's a little bit of complexity here that I'm not dealing with:

- We probably shouldn't send an invitation to a user until we've saved the request. 
- But the request is invalid without a `User` as the requestee - so validation will fail if we move this later on.

I think the answer is probably conditional validation on the `Request` requirement for a requestee